traffic_portal/package.json

```diff
@@ -1,13 +1,14 @@
 {
   "name": "traffic_portal",
   "devDependencies": {
+    "browserify": "^16.5.1",
     "connect-livereload": "0.6.1",
     "connect-modrewrite": "0.10.2",
     "connect-timeout": "1.9.0",
     "express": "4.17.1",
     "grunt": "1.0.4",
     "grunt-adam-compass": "0.7.1",
-    "grunt-browserify2": "0.1.8",
+    "grunt-browserify": "^5.3.0",
```

Comment:
this always worries me. see how everything else is pinned to a version? i got burned in the past when not pinning to a version.....then again that's a good way to fall behind on libraries. but i suppose our ci pipieline is better now and should catch any problems as a result of library upgrades.

Reply:
Good point, perhaps `~` (patch releases only) would be better. In any case, I'll make it exact versioning for now as that change should probably be made to the entire file in a separate PR.